src/Supply/Application/Service/InventoryImporter.php

```diff
@@ -138,4 +139,9 @@ private function classifyCampaign(
             }
         }
     }
+
+    public function clearInventoryForHost(string $host): void
```

Comment:
I think that this function should be used in two other places where `$this->markedCampaignsAsDeletedService` is called

Reply:
changed